lib/apiservers/service/restapi/handlers/vch_list_get.go

```diff
@@ -98,29 +98,39 @@ func listVCHs(op trace.Operation, d *data.Data, validator *validate.Validator) (
 func vchsToModels(op trace.Operation, vchs []*vm.VirtualMachine, executor *management.Dispatcher) []*models.VCHListItem {
 	installerVer := version.GetBuild()
 	payload := make([]*models.VCHListItem, 0)
```

Comment:
`var payload []*models.VCHListItem` should be enough. A Nil slice behaves like an empty slice.

Reply:
This list of length 0 is a placeholder for the json field in API response body so that it's not null.

If I don't allocate a list of length 0, the json output would look like this:
```
{
    "vchs": null
}
```

and we prefer an empty list instead of null:
```
{
    "vchs": []
}
```